src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -91,6 +92,7 @@ public SpriteMapView(GameInstance game)
     menuArtist = new MenuArtist(game, this);
 
     myGame = game;
+    COMPREHENSIVE_HUD_H_SIZE = SpriteLibrary.getCoOverlay(myGame.activeArmy.cos[0], true).getWidth() + 42;
```

Comment:
Why+42? I know you love your magic 42s, but you know I hate magic.

Reply:
I get it was there before, and it's just a buffer for good measure, so not the end of the world, but descriptive words are nice.